packages/cascara/src/ui/Pagination/Pagination.js

```diff
@@ -169,12 +171,12 @@ const Pagination = ({ totalRecordCount, state }) => {
                   type='button'
                 >
                   <i className='right chevron icon' />
-                </button>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
+                </Button>
+              </Role>
+            </Role>
+          </Role>
+        </Role>
+      </Role>
```

Comment:
@rociobaena I like that we are swapping all those **divs** by **Role** components, was there any reason not to limit us to only replace the root wrapper? I think it makes sense to have them all replaced.

Reply:
Manu, I have doubts about this change too, that's why I added the comment to put special attention to this component. I added the Role to all these components because they are containers at the end. I know that piece of code is not visible enough, but when I saw this, it is a big container containing two more components, that's why I decided to use Role here too. 
![image](https://user-images.githubusercontent.com/87443054/141237845-b8efa0c6-af1a-406c-96d7-f5fbbc8439d4.png)